pkg/maincmd/maincmd.go

```diff
@@ -89,7 +89,7 @@ func Main(args []string, stdin io.Reader, stdout io.Writer, stderr io.Writer, cf
 	}
 
 	if !opts.Daemon {
-		return fmt.Errorf("NYI: client mode")
```

Comment:
I was confused when I tried to start the binary, I didn't even realize this was error message. I had to do a web search to learn what NYI means

Reply:
Guilty as charged :) I tend to type NYI quickly when I’m prototyping code.